app/server/lib/SQLiteDB.ts

```diff
@@ -258,6 +258,10 @@ export class SQLiteDB implements ISQLiteDB {
   private constructor(protected _db: MinDB, private _dbPath: string) {
   }
 
+  public async interrupt(): Promise<void> {
+    return this._db.interrupt?.();
```

Comment:
Wondering if it is ok for the `interrupt` method to be missing. If that's indeed possible in some configurations, should the SQL endpoint still be enabled in those cases?

Reply:
I made this more explicit, and also added a check that the wrapper had been reviewed for how it handles multiple statements.